nautobot_ssot/integrations/dna_center/diffsync/models/nautobot.py

```diff
@@ -23,12 +23,19 @@
 )
 
 try:
-    import nautobot_device_lifecycle_mgmt  # noqa: F401
+    from nautobot_device_lifecycle_mgmt import SoftwareLCM  # noqa: F401
 
     LIFECYCLE_MGMT = True
 except ImportError:
     LIFECYCLE_MGMT = False
 
+try:
+    from nautobot.dcim.models import SoftwareImageFile, SoftwareVersion  # noqa: F401
+
+    SOFTWARE_VERSION_FOUND = True
```

Comment:
Should this variable be called something like `SOFTWARE_VERSION_FOUND_IN_CORE` or something? Because given `LIFECYCLE_MGMT` it would be there up until some version so it may be good to differentiate these.

Reply:
Sure, updated.